README.md

```diff
@@ -17,3 +17,48 @@ All gen-txs have been merged.
 
 The genensis file will be release at 06:00 UTC on Dec 11th.
 
+## Instructions for Launching Game of Stakes
+
+
+### Welcome to the Launch of Game of Stakes.
+
+You are part of large experiment in computer science. Today we will be launching the largest Byzantine Fault Tolerant network in history.
```

Comment:
You are _about to be_ part of _the largest_ experiment in computer science. Today _you are going to help_ launch the largest Byzantine Fault Tolerant network in history.